src/applications/personalization/preferences/helpers.jsx

```diff
@@ -383,7 +383,7 @@ export const benefitChoices = [
     title: 'Careers and Employment',
     description:
       'Find a job, build skills, or get support for my own business.',
-    slug: 'careers',
+    code: 'careers-employment', // TODO: update rest
```

Comment:
We can probably remove these TODO comments, as the content has been updated.

Reply:
I'll go ahead and remove these now.